src/test/skript/tests/misc/pull-7129-pretty-quotes.sk

```diff
@@ -0,0 +1,30 @@
+
+command /prettyquotecommandtest “prettyquoteliteral” <text>:
+	trigger:
+		set {-PrettyQuoteArg} to arg
```

Comment:
This - stuff isn't a thing in the testing environment so you need to actually delete the variable when the test is done instead.